src/core/thread/csl_tx_scheduler.cpp

```diff
@@ -260,7 +261,7 @@ void CslTxScheduler::HandleSentFrame(const Mac::TxFrame &aFrame, Error aError)
     HandleSentFrame(aFrame, aError, *child);
 
 exit:
-    return;
+    RescheduleCslTx();
```

Comment:
@abtink wouldn't this mean we would be calling RescheduleCslTx() twice for some scenarios (in case of any error RescheduleCslTx() is also getting called from CslTxScheduler::HandleSentFrame)?

Reply:
This PR is removing the other call to `RescheduleCslTx()` (inside the `switch()` under error case). Line 315 removed. Basically moving it here at `exit` label (so it is called under all cases).

Reply:
Got it, missed that change. Thanks!